chain-connect/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@gala-chain/connect",
-  "version": "1.4.2",
+  "version": "1.4.3",
```

you don't need to provide this, next release version is `1.4.2`, and our CI creates automatic PRs with version bump after release, for instance: https://github.com/GalaChain/sdk/pull/350/files

```suggestion
  "version": "1.4.2",
```